mesmerize_napari/tests/test_core.py

```diff
@@ -218,7 +217,18 @@ def test_cnmf():
     df = load_batch(batch_path)
     print(df)
     # Confirm output path is as expected
-
-#
-# def test_remove_item():
-#     pass
\ No newline at end of file
+    assert df.iloc[-1]['outputs']['success'] is True
+    assert df.iloc[-1]['outputs']['traceback'] is None
+    assert Path(os.path.join(vid_dir,
+        f'{df.iloc[-1]["uuid"]}_cnmf-memmap__d1_60_d2_80_d3_1_order_C_frames_2000_.mmap')) == \
+           get_full_data_path(df.iloc[-1]['outputs']['cnmf-memmap-path']) == \
+          Path(os.path.join(vid_dir, df.iloc[-1]['outputs']['cnmf-memmap-path']
+                       ))
+
+    assert Path(os.path.join(vid_dir,
+        f'{df.iloc[-1]["uuid"]}.hdf5')) == \
+        get_full_data_path(df.iloc[-1]['outputs']['cnmf-hdf5-path']) == \
+        Path(os.path.join(vid_dir, df.iloc[-1]['outputs']['cnmf-hdf5-path']))
```

Comment:
also test all the cnmf extensions to make sure the output is the same as what we get for the test dataset